src/modules/vendor/pages/ni/users/companies/CompanyProfile.vue

```diff
@@ -25,13 +35,26 @@ export default {
   },
   data () {
     return {
+      companyTradeName: '',
       tabsContent: [{ label: 'Infos', name: 'infos', default: this.defaultTab === 'infos', component: ProfileInfo }],
     }
   },
   computed: {
     ...mapState('company', ['company']),
-    companyName () {
-      return get(this.company, 'name') || '';
+    companyType () {
+      switch (get(this.company, 'type')) {
+        case COMPANY:
+          return 'Entreprise';
+        case ASSOCIATION:
+          return 'Association';
+        default:
+          return '';
+      };
+    },
```

Comment:
Vu que tu renvoie juste une string, j'aurais plutot mis dans les data (ou dans les constants?), un objet `companyTypes` et j'aurais renvoyé `companyTypes[get(this.company, 'type')] || ''`

Reply:
tu peux mutualiser avec a `companyTypeOptions` du repertoire structure